src/utils/zip.ts

```diff
@@ -95,4 +95,45 @@ export class Zip {
             archive.finalize();
         });
     }
+
+    public static zipFile(filePath: string, destinationPath: string = null, extension = null): Promise<void> {
```

Comment:
Please, consider use `zip` instead of creating a new function. Both are doing the same. You might need refactor the function in order to allow compress just a file